vagrant/guest/bin/m2setup.sh

```diff
@@ -76,6 +76,13 @@ for arg in "$@"; do
                 exit -1
             fi
             ;;
+        --proj-version=*)
+            PROJ_VERSION="${arg#*=}"
+            if [[ ! "$PROJ_VERSION" =~ ^[a-z0-9\.-]+$ ]]; then
```

Comment:
Can you add an `*` to the allowed character list as well? Ex: 2.0.* can be used to install latest 2.0 patch release. If we have to use this after 2.1 becomes the latest stable release, having the wildcard allowed will be very handy.